### ISSUE TITLE
Future Forever

### ISSUE THREAD

**Issue by Alexander1414x** [foreversd/forever#1094]
Hello
please future fucntion forever  to use :

- [ ] forever --delay 15 index.js - restart your file for re-connect.

- [ ] forever --auto - function , auto detect when file your using to go start.

- [ ] forever --auto --delay 15 - auto restart your file from after re-connect file and re-detect again .

- [ ] forever --monitoring -delay 15 , deep active from your restart file and re-connect from VPS\VDS again.

- [ ] forever --auto -save-file - auto save file for your really time coding about, nodemon your restarting and save your file from your coding.

**Comment by framerate** [comment]
@kibertoad this is actually news to me, is there info on this anywhere? What is the alternative people are using these days?

Thanks!

**Comment by kibertoad** [comment]
@framerate Yes. Literally the first paragraph in the readme :). For modern Dockerized production environments you are actually encouraged not to use anything: https://github.com/goldbergyoni/nodebestpractices#-83-let-the-docker-runtime-handle-replication-and-uptime
For development needs nodemon is a better alternative.

**Comment by framerate** [comment]
Oh goodness. I missed that line, thank you. I'm used to seeing big alerts at top of readme warning of deprecation I guess :(

Thanks for the best practices readme. I've never seen that before. Been around a long time and still using what always worked but just learned or forevers death. 

Thanks!
